### PR TITLE
sig-ddl: promote 'blacktear23' to active contributor

### PR DESCRIPTION
**Pull request by bb7133**
@blacktear23 is very active in the past few months, his contributions
includes:

- [14 merged
PRs](https://github.com/pingcap/parser/pulls?q=is%3Apr+author%3Ablacktear23+is%3Aclosed) in TiDB repo and 10 of them are
DDL-related
- [2 merged PRs](https://github.com/pingcap/parser/pulls?q=is%3Apr+author%3Ablacktear23+is%3Aclosed) in parser
repo

### CODE REVIEW

**Review by wjhuang2016 — COMMENTED** [review]
LGTM

**Review by AilinKid — COMMENTED** [review]
LGTM

**Comment by hi-rustin** [comment]
> LGTM

About the new robot command you can refer to https://prow.tidb.io/plugins?repo=pingcap%2Fcommunity.
The design doc: https://book.prow.tidb.io/plugins/lgtm.html

cc: @wjhuang2016  @AilinKid

File: special-interest-groups/sig-ddl/membership.json [hunk 1]
```diff
@@ -71,6 +71,9 @@
     },
     {
       "githubName": "zhaox1n"
-    }
+    },
+	{
```
Comment (hi-rustin) [inline special-interest-groups/sig-ddl/membership.json:75]: Better format it.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

**Comment by hi-rustin** [comment]
/hold 

Waiting fmt the JSON file.

File: special-interest-groups/sig-ddl/membership.json (hunk 1 above)
Comment (bb7133) [reply special-interest-groups/sig-ddl/membership.json:75]: Done, PTAL

**Comment by hi-rustin** [comment]
/hold cancel

/assign @wjhuang2016 

You can `/merge` it now.

**Review by AilinKid — COMMENTED** [review]
LGTM

**Pull request merged**